### ISSUE TITLE
Create login page

### ISSUE THREAD

**Issue by AlexZhidkov** [AlexZhidkov/wil-business#2]
When not signed-in user clicks on Interested button, he should be forwarded to the Login page and after successful login with Google redirected to corresponding EOI page.

**Comment by AlexZhidkov** [comment]
Not signed in users who sign in from /business page assigned the role "isBusiness", from /student page - "isStudent".

**Comment by sbhat423** [comment]
Will be changing next, the routes, so that signed in user cannot go to pages for different role, using url.

**Comment by AlexZhidkov** [comment]
> Will be changing next, the routes, so that signed in user cannot go to pages for different role, using url.

@sbhat423 I don't think we need that. Even more, we can allow the same user to belong to Student and Business roles. But please don't do it now.

**Comment by sbhat423** [comment]
Sure! I might have to tweak 'role' property a little bit to support multiple roles.

**Comment by AlexZhidkov** [comment]
> Sure! I might have to tweak 'role' property a little bit to support multiple roles.
I've done that.
